src/demos/calcite-button.html

```diff
@@ -352,17 +351,24 @@ <h3>Scales</h3>
   <calcite-button title="s scale button" scale='s'>s scale button</calcite-button>
   <calcite-button title="xs scale button" scale='xs'>xs scale button</calcite-button>
 
+  <h3>Scales with icons</h3>
+  <calcite-button title="xl scale button" icon="check-circle" scale='xl'>xl scale button</calcite-button>
+  <calcite-button title="l scale button" icon="check-circle" scale='l'>l scale button</calcite-button>
+  <calcite-button title="m / default scale button" icon="check-circle">m scale button</calcite-button>
```

Comment:
Nitpick: `m (default) ...`? Related, does the text in this button also need to mention it's the default?